Libraries/libcar/Sources/AttributeList.cpp

```diff
@@ -99,3 +99,24 @@ operator()(AttributeList const &attributes) const
     /* Not a great hash. */
     return attributes.count();
 }
+
+
+std::vector<uint8_t> AttributeList::
+Write(size_t count,
+    uint32_t *identifiers) const
```

Comment:
Hard to see from this diff, but I was sticking with the style that was there, but I agree, it can go on one line. I'll clean it up.